bin/web.js

```diff
@@ -17,6 +20,36 @@ if (process.env.ANALYTICS_TOKEN) {
     analytics = new Analytics(process.env.ANALYTICS_TOKEN);
 }
 
+// Set up for https termination
+var key = "", cert = ""
+
+if (process.env.HTTPS_KEYFILE !== 'undefined') {
+    try {
+        key = fs.readFileSync(process.env.HTTPS_KEYFILE);
```

Comment:
i would just throw here in any case, if the env var is set and there's an error reading the file, fail hard